gateway_code/utils/CC2538.py

```diff
@@ -38,7 +38,8 @@ class CC2538(object):
     DEBUG = ('')
 
     @property
-    def add_command(self):
+    def addr_command(self):
+        #TODO Do it a better way with python
```

Comment:
There's a TODO here.
Could be better to replace this long and complex shell command above by python code.